opal/mca/btl/openib/btl_openib_mca.c

```diff
@@ -291,6 +292,19 @@ int btl_openib_register_mca_params(void)
     if (0 > tmp) ret = tmp;
     OBJ_RELEASE(new_enum);
 
+    /*
+     * Provide way for using to override policy of ignoring IB HCAs
+     */
+
+    mca_btl_openib_component.allow_ib = false;
+    tmp = mca_base_component_var_register(&mca_btl_openib_component.super.btl_version,
+                                          "allow_ib",
+                                          "Override policy since Open MPI 4.0 of ignoring IB HCAs for openib BTL",
+                                          MCA_BASE_VAR_TYPE_BOOL, NULL,
+                                          0, 0, OPAL_INFO_LVL_9,
```

Comment:
Minor: per https://github.com/open-mpi/ompi/wiki/MCAParamLevels, I'd probably make this level 5.